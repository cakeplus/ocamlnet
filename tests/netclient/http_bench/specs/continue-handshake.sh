. helpers.sh

start_test_server -line 6 -file data/status-100 -line 7 -file data/framed
trap "stop_test_server" EXIT
request -handshake -put-small / -run
