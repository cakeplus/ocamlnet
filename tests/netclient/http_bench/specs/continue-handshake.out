22F1A22D055CDBCC183A9FC3FCCA7732
